### ISSUE TITLE
[bitnami/argo-cd]: "Unable to load data: configmap "argocd-tls-certs-cm" not found"

### ISSUE THREAD

**Issue by aurelioacn** [bitnami/charts#13400]
### Name and Version

bitnami/argocd

### What steps will reproduce the bug?

Deploy bitnami/argocd helm chart without set .Values.config.tlsCerts.
Application is deployed successfully.
From ArgoCD interface navigate to Settings -> Certificates.

### Are you using any custom parameters or values?
```yaml
controller:
    podSecurityContext:
        enabled: false
    containerSecurityContext:
        enabled: false
        
server:
    insecure: true
    podSecurityContext:
        enabled: false
    containerSecurityContext:
        enabled: false

repoServer:
    podSecurityContext:
        enabled: false
    containerSecurityContext:
        enabled: false

config:
    secret:
        argocdServerAdminPassword: "1234ABCD"
    extraKnownHosts: |  
           github.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEmKSENjQEezOmxkZMy7opKgwFB9nkt5YRrYMjNuG5N87uRgg6CLrbo5wAdT/y6v0mKV0U2w0WZ2YB/++Tpockg=        

redis:
    master:
        podSecurityContext:
            enabled: false
        containerSecurityContext:
            enabled: false
```
### What is the expected behavior?

Since config.tlsCerts parameter is not mandatory the deployed ArgoCD should not complain about configmap "argocd-tls-certs-cm" not begin created and users should be able to add certificates from the interface.

### What do you see instead?

Pop up "Unable to load data: configmap "argocd-tls-certs-cm" not found" and user cannot add manually any certificates because this ConfigMap is not created.

### Additional information

the argocd-tls-certs-cm is not created because of this condition: 
https://github.com/bitnami/charts/blob/main/bitnami/argo-cd/templates/tls-certs-cm.yaml#L1

If I add the parameter config.tlsCerts a dummy certificate, the argocd-tls-certs-cm is created as expected and there are no issues and the users can add certificates from interface with no problem.

**Comment by javsalgar** [comment]
Hi,

This is strange, as the value is empty, right? Could you confirm that you are not setting that value? Also, which helm version are you using?

**Comment by anetoATOS** [comment]
I config that when config.tlsCerts is empty the issue happens.

This is the chart Im using:

apiVersion: v2
appVersion: 2.4.8
dependencies:
- condition: redis.enabled
  name: redis
  repository: https://charts.bitnami.com/bitnami
  version: 17.x.x
- name: common
  repository: https://charts.bitnami.com/bitnami
  tags:
  - bitnami-common
  version: 1.x.x
sources:
- https://github.com/bitnami/containers/tree/main/bitnami/argo-cd
- https://github.com/argoproj/argo-cd/
- https://github.com/bitnami/containers/tree/main/bitnami/dex
- https://github.com/dexidp/dex
version: 4.0.6

**Comment by aurelioacn** [comment]
@corico44 the other way, 
I would propose to change what triggers this error "Unable to load data: configmap "argocd-tls-certs-cm" not found" when this cm is not created https://github.com/bitnami/charts/blob/main/bitnami/argo-cd/templates/tls-certs-cm.yaml

At the end if I have this cm created but with dummy values there are no issues so this file existing or not is not really a must for the application.

If this proposal cannot be archived then I would do what you mentioned and always create the cm with or without user informing tlsCerts.

**Comment by corico44** [comment]
Okay @aurelioacn , I understand what you mention. If you'd like, you can contribute to the chart by opening a PR and we will review it. [Here](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#adding-a-new-chart-to-the-repository) you can find the requirements and scaffolding to create a Helm Chart following the Bitnami guidelines.

**Comment by corico44** [comment]
Since this error is considered an important feature to fix, I have created a task internally to review this error. We currently have a lot on our plate. For this reason, we will notify you in this same issue when the error has been solved.

**Comment by Nello-Angelo** [comment]
https://github.com/bitnami/charts/issues/15736

**Comment by CeliaGMqrz** [comment]
Hi @aurelioacn, 

Sorry for the delay. It has been resolved in [#15754](https://github.com/bitnami/charts/pull/15754).

Thanks for your feedback!
